### ISSUE TITLE
Different bar configuration on different pages

### ISSUE THREAD

**Issue by hsoeg** [AdyRock/com.ady.button_plus#113]
Hi

I'm currently setting up the button plus using the homey app. What I seem to be missing is the ability to add different bars to different pages. If I go into the web setup, I can do it - but when configuring through homey it seems I can't. Or am I missing something?

Kind regards 
Henrik

**Comment by AdyRock** [comment]
That has only just been added to the new firmware and will be in the next Homey app update. I will probably release a test version this evening.

**Comment by hsoeg** [comment]
ok - thanks. I'm all new to this, so just went with the latest version of everything. Have a nice evening ;)

**Comment by AdyRock** [comment]
I have just published the test version, but as soon as I hit the button, I realised there is a flaw in the implementation. The new version will allow you to define which page the button appears on but not allow multiple pages, so I need to do a bit more work on that.

**Comment by hsoeg** [comment]
Ok. Thanks - I Will await your revised version.Btw, is it possible to hvide the ip adresse shown in the display?Rgds HenrikSendt fra min iPadDen 23. dec. 2024 kl. 18.11 skrev Adrian Rockall ***@***.***>:﻿
I have just published the test version, but as soon as I hit the button, I realised there is a flaw in the implementation. The new version will allow you to define which page the button appears on but not allow multiple pages, so I need to do a bit more work on that.

**Comment by AdyRock** [comment]
If you open up the Advanced settings of the device and change the Status bar to Bottom, it will show the IP address.

**Comment by hsoeg** [comment]
Yes, but it seems that every time I set it to none in the advanced interface, and afterwards make an update in the homey interface - it is brought back...

**Comment by hsoeg** [comment]
Any news on the original request for supporting different bars to different pages?

**Comment by AdyRock** [comment]
It should be working in the latest test version.

**Comment by hsoeg** [comment]
Great!

When do you expect to release it to production?

**Comment by AdyRock** [comment]
Once I am as sure as possible that it's stable. 
I have a couple of reports to look into, but I am struggling to find time ATM. 
Hopefully, I can knock them off over the weekend.
